UPGRADING.INTERNALS

```diff
@@ -62,6 +62,7 @@ PHP 8.5 INTERNALS UPGRADE NOTES
 
 - ext/libxml
   . The refcount APIs now return an `unsigned int` instead of an `int`.
+  . Removed php_libxml_xmlCheckUTF8(). Use xmlCheckUTF8() from libxml instead.
```

Comment:
Do we really not have an exported UTF-8 checking function ourselves?! I checked and couldn't find anything.